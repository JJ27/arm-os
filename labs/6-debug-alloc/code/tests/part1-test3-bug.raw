FOUND: </dev/ttyUSB0>
opened tty port </dev/ttyUSB0>.
pi-install: tty-usb=</dev/ttyUSB0> program=<./tests/part1-test3-bug.bin> about to boot
TRACE:simple_boot: sending 13160 bytes, crc32=32ac01c1
waiting for a start
putting code
bootloader: Done.
listening on ttyusb=</dev/ttyUSB0>
redzone checking: init
test3: use after free
TRACE:ck_heap_errors:going to check heap
TRACE:mem_check:ERROR:Freed block 1 [0x17fef8] corrupted at offset 0
TRACE:hdr_print:	logical block id=1, [addr=0x17fef8] nbytes=4
TRACE:hdr_print:	Block allocated at: tests/part1-test3-bug.c:notmain:9
TRACE:check_list:	Wrote block after free!
TRACE:ck_heap_errors:checked 1 blocks, detected 1 errors
TRACE:notmain:SUCCESS: detected corruption in 1 [0x17fef8]
DONE!!!

Saw done

bootloader: pi exited.  cleaning up
