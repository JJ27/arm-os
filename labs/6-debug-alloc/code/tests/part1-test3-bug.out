TRACE:ck_heap_errors:going to check heap
TRACE:mem_check:ERROR:Freed block 1  corrupted at offset 0
TRACE:hdr_print:	logical block id=1,  nbytes=4
TRACE:hdr_print:	Block allocated at: tests/part1-test3-bug.c:notmain:9
TRACE:check_list:	Wrote block after free!
TRACE:ck_heap_errors:checked 1 blocks, detected 1 errors
TRACE:notmain:SUCCESS: detected corruption in 1 
